src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -177,13 +177,18 @@ if string.find( _HOST, "ComputerCraft" ) == 1 then
     end
 end
 
+local nativeyield = coroutine.yield
+function coroutine.yield( sFilter )
+    return nativeyield( sFilter )
+end
+
```

Comment:
Why override coroutine.yield itself??? Is that not simply overhead without any benefit?